BUILD_INSTRUCTIONS_FOR_LINUX.md

```diff
@@ -0,0 +1,39 @@
+# Build Instructions for Linux
+
+## Required Libraries
+*   Python2
+*   TensorFlow
+*   Git
+*   Bazel-3.7.2
+
+
+## Installation Instructions
+
+#### For Python2
+In the console, run:
+```bash
+sudo apt install python
+```
+
+#### For TensorFlow
+In the console, run:
+```bash
+sudo apt install python3-pip
+pip install tensorflow
+```
+
+#### For Git
+In the console, run:
+```bash
+sudo apt-get update
+sudo apt-get install git
+```
+
+#### For Bazel
```

Comment:
Thanks for the contribution, @milindthakur177 !

I would recommend using bazelisk here rather than installing Bazel directly. Could you help try that and update?

Reply:
Sure, I'll try bazelisk and update the instructions. I installed bazel directly since the installation instructions are quite straightforward.